packages/manager/apps/pci-rancher/src/data/hooks/useFormattedPrices/useFormattedPrices.tsx

```diff
@@ -10,8 +11,9 @@ export const useFormattedRancherPrices = (
   plans: RancherPlan[],
   pricing?: TRancherPricing[],
 ) => {
-  const { getFormattedHourlyCatalogPrice } = useCatalogPrice(5);
-  const { getFormattedMonthlyCatalogPrice } = useCatalogPrice(2);
+  const { getFormattedCatalogPrice: hourly } = useCatalogPrice(5);
+  const { getFormattedCatalogPrice: monthly } = useCatalogPrice(2);
+  const { t } = useTranslation(['order-price']);
```

Comment:
```suggestion
  const { t } = useTranslation('order-price');
```

If not backup translation, we can just use a string